mne/viz/_3d.py

```diff
@@ -415,6 +454,19 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    limits : str | 3-tuple | dict
```

Comment:
I think I'll collapse the tuple functionality into the dict. Then `clim` can be `'auto'` or `dict` to make it clearer but leave the option to easily have the function determine automatic limits.